tcms/core/contrib/auth/tests.py

```diff
@@ -105,8 +105,8 @@ def test_open_registration_page(self):
             html=True)
 
     @patch('tcms.core.contrib.auth.models.sha1')
-    def assert_user_registration(self, username, sha1, follow=False):
-        sha1.return_value.hexdigest.return_value = self.fake_activate_key
+    def assert_user_registration(self, username, sha_1, follow=False):
+        sha_1.return_value.hexdigest.return_value = self.fake_activate_key
```

Comment:
Convert the patch decorator to a context manager, see:
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

and remove the disable=no-value-for-parameter switches below.

A context manager is more readable in this case. Frankly it took me a while to spot the decorator and figure out why we call this method without actually passing the `sha1` parameter.